From 5c21196950e7cd4fc3a59cfc66b619f3e037bba3 Mon Sep 17 00:00:00 2001
From: miwu04 <me@alanlin.icu>
Date: Thu, 16 Apr 2026 21:47:51 +0800
Subject: [PATCH 4/4] Revert "chore: restrict supported architectures to
 current OS and specific CPU types in pnpm-workspace.yaml"

This reverts commit 3359b2a3e3561529c356e2b7d5ddcea51fa21ea1.
---
 pnpm-workspace.yaml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/pnpm-workspace.yaml b/pnpm-workspace.yaml
index 9eff6408..0c1884ab 100644
--- a/pnpm-workspace.yaml
+++ b/pnpm-workspace.yaml
@@ -16,6 +16,10 @@ minimumReleaseAgeExclude:
 supportedArchitectures:
   os:
     - current
+    - win32
+    - darwin
+    - linux
   cpu:
+    - current
     - x64
     - arm64
-- 
2.52.0

